src/mame/drivers/seta.cpp

```diff
@@ -10468,6 +10545,9 @@ ROM_START( thunderl )
 	ROM_REGION( 0x100000, "x1snd", 0 )  /* Samples */
 	ROM_LOAD( "r28", 0x000000, 0x080000, CRC(a043615d) SHA1(e483fa9fd8e922578a9d7b6ced0750643089ca78) )
 	ROM_LOAD( "r27", 0x080000, 0x080000, CRC(cb8425a3) SHA1(655afa295fbe99acc79c4004f03ed832560cff5b) )
+
+	ROM_REGION(0x200, "plds", 0)        /* Protection */
+	ROM_LOAD("tl-9", 0x000000, 0x117, CRC(3b62882d) SHA1(a590648cb013f20d837f18ddb2e839a89bac5fcb))
```

Comment:
Please flag as `BAD_DUMP` if this is a recreation based on observation of behaviour.

Reply:
In fact, the original PAL is protected and cannot be read. I bruteforce it to extract the logic equations.
Even this, if it's still needed to add BAD_DUMP, let me know and I'll add it

Reply:
We tend to use `BAD_DUMP` when we don’t actually know the equations and add a comment to indicate that it’s a recreation.  It may be less than ideal, but it’s the best we can do for now.

Reply:
Done!